doc/integrator/editing.rst

```diff
@@ -0,0 +1,49 @@
+.. _integrator_editing:
+
+Editing (with CGXP)
+===================
```

Comment:
Should be in Legacy section

Reply:
In fact, this was already included only in Legacy section. But I will rename the file to make it more clear.

Reply:
+1